jobs/job_utils.py

```diff
@@ -241,79 +219,34 @@ def get_beam_query_from_ndb_query(query):
 
     Args:
         query: datastore_services.Query. The NDB query to convert.
+        namespace: str|None. Namespace for isolating the NDB operations of
+            tests.
 
     Returns:
         beam_datastore_types.Query. The equivalent Apache Beam query.
     """
     kind = query.kind
-    namespace = query.namespace
-    project = query.app
+    namespace = namespace or query.namespace
+    project = query.project or feconf.OPPIA_PROJECT_ID
```

Comment:
Why can't this just always be feconf.OPPIA_PROJECT_ID?

Reply:
@brianrodri PTAL

Reply:
Not sure here, I think it can be removed but not sure if it can lead to errors in the future.

Reply:
Probably remove for now and reinstate later if needed?

Reply:
Done.